policyengine_us/parameters/gov/states/al/tax/income/exemptions/dependent.yaml

```diff
@@ -0,0 +1,27 @@
+description: Alabama provides dependent exemption amounts based on the filed AGI amount.
+
+brackets:
+- threshold:
+      2022-01-01: -.inf
+  amount: 
+      2022-01-01: 1_000
+- threshold:
+      2022-01-01: 50_001
+  amount: 
+      2022-01-01: 500
+- threshold:
+      2022-01-01: 100_001
+  amount: 
+      2022-01-01: 300
+
+metadata:
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  threshold_period: year
+  type: single_amount
+  label: Alabama dependent exemption
+  reference:
+  - title: 2022 Form 40 Alabama Income Tax Return
+    href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/02/22f40.pdf
```

Comment:
I'm not seeing this structure in this form, could you link the instruction booklet or leave a comment with the page number if I missed it?